BranchUnityTestBed/Packages/manifest.json

```diff
@@ -44,6 +44,16 @@
     "com.unity.modules.video": "1.0.0",
     "com.unity.modules.vr": "1.0.0",
     "com.unity.modules.wind": "1.0.0",
-    "com.unity.modules.xr": "1.0.0"
-  }
-}
+    "com.unity.modules.xr": "1.0.0",
+    "com.google.external-dependency-manager": "1.2.179"
```

Comment:
So to confirm, the binaries were bundled in but now should be resolved by this package manager.

Reply:
In other words, this is what clients will now have to add in their integration

Reply:
Our SDK binaries were not bundled in. What was bundled in was the tool to fetch them, external dependency manager for unity. This is now what the client must install on their own.